products/facilities/metadata.yml

```diff
@@ -0,0 +1,303 @@
+    in the source data or provided by GeoClient.
+  data_type: double
+  example: '213839.74968230928'
+- name: bin
+  display_name: bin
+  description: >
+    The BIN values for the buildings the facility is located in. This
+    field contains blanks, because 1) some lots do not have buildings on
+    them, and 2) some lots have multiple buildings and a single BIN could not be
+    reliably assigned given the information provided in the source data.
+  data_type: text
+  example: '1083624.0'
+- name: bbl
+  display_name: bbl
+  description: >
+    The BBL values for the tax lots the facility is located on. This field
+    contains blanks, because some facility categories, like pedestrian plazas
+    are not located on tax lots.
+  data_type: text
+  example: '1007860001.0'
+- name: commboard
+  display_name: community board
+  description: >
+    The Community District the facility is within. This field
+    contains blanks because the Community District boundaries do not capture facilities
+    beyond the shoreline.
+  data_type: integer
+  example: '105'
+- name: council
+  display_name: council district
+  description: The City Council District the facility is within
+  data_type: integer
+  example: '1'
+- name: censtract
+  display_name: census tract
+  description: The U.S. Census Tract the facility is within
+  data_type: integer
+  example: '2100'
+- name: nta
+  display_name: nta
+  description: >
+    The Neighborhood Tabulation Area (NTA) the facility is within. This
+    field contains blanks because the NTA boundaries do not capture facilities
+    beyond the shoreline.
+  data_type: text
+  example: MN17
+- name: facgroup
+  display_name: facgroup
+  description: The value representing the group type, the second broadest category.
+  data_type: text
+  example: DAY CARE AND PRE-KINDERGARTEN
+- name: facsubgrp
+  display_name: facsubgrp
+  description: The value representing the subgroup type.
+  data_type: text
+  example: PARKING LOTS AND GARAGES
+- name: factype
+  display_name: factype
+  description: >
+    The value representing the facility type, the most granular category
+    of facilities.
+  data_type: text
+  example: SUMMER ONLY FEEDING SITE
+- name: capacity
+  display_name: capacity
+  description: >
+    How many of capacity type/unit the facility is intended to hold. For
+    many types of facilities, no capacity information was available. The capacity-related
+    fields are blank when no information was provided in the source data.
+  data_type: text
+  example: '200.0'
+- name: captype
+  display_name: captype
+  description: The value representing the unit type of capacity, such as beds, visitors, seats, etc.
+  data_type: text
+  example: seats
+  values:
+  - [seats, ""]
+  - [beds, ""]
+  - [routes, ""]
+- name: optype
+  display_name: optype
+  description: Indicates whether the operating entity is public or non-public.
+  data_type: text
+  example: Non-public
+  values:
+  - [Non-public, ""]
```

Comment:
These picklist values are generated by the scraper when it sees fewer than (IIRC) 10 or so values in the column metadata. The format of the picklist values is a two element list, with the raw value, and the human readable variety. In the cases where the raw value matches the description, I've just left them blank. 

A different case might be boro code, where it's a number, but the description is the boro name itself. e.g. "1" -> "Manhanttan"